.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug Report"
```

Comment:
What are these: `U0001F41B`? Could it be removed? @acocac

Reply:
They refer to the emoticon next to the issue type.